src/app/components/core/configuration-manager/static-configuration/static-configuration.component.html

```diff
@@ -36,7 +36,7 @@
             class="readonly-editor" [disabled]="true">
           </ngx-codemirror>
         </ng-container>
-        <span *ngSwitchDefault class="help p-1 m-0">{{configuration.value ? configuration.value : "None"}}</span>
```

Comment:
What was the issue with p-1 m-0? Just want to make sure, We know why are we removing this.

Reply:
This was only for one type of control and it was causing alignment issue on configuration page. I don't see any need of it so I removed.